RELEASENOTES.md

```diff
@@ -1,18 +1,40 @@
 # Release Notes
 
-![Ripple](docs/images/ripple.png)
+![XRP](docs/images/xrp-text-mark-black-small@2x.png)
 
-This document contains the release notes for `rippled`, the reference server implementation of the Ripple protocol. To learn more about how to build and run a `rippled` server, visit https://ripple.com/build/rippled-setup/
+This document contains the release notes for `rippled`, the reference server implementation of the Ripple protocol. To learn more about how to build, run or update a `rippled` server, visit https://xrpl.org/install-rippled.html
 
-> **Do you work at a digital asset exchange or wallet provider?** 
-> 
-> Please [contact us](mailto:support@ripple.com). We can help guide your integration.
+ 
+Have new ideas? Need help with setting up your node? Come visit us [here](https://github.com/ripple/rippled/issues/new/choose)
 
-## Updating `rippled`
+# Releases
 
-If you are using Red Hat Enterprise Linux 7 or CentOS 7, you can [update using `yum`](https://ripple.com/build/rippled-setup/#updating-rippled). For other platforms, please [compile from source](https://wiki.ripple.com/Rippled_build_instructions).
+## Version 1.5.0
+
+The `rippled` 1.5.0 release introduces several improvements and new features, including support for gRPC API, API versioning, UNL propagation via the peer network, new RPC methods `validator_info` and `manifest`, augmented `submit` method, improved `tx` method, improved CLI parsing, improved protocol-level handshaking protocol, improved package building and various other minor bug fixes and improvements.
```

Comment:
"improved protocol-level handshaking protocol" reads oddly.  I'm also not at all sure what it means, so I can't suggest an improvement.

Does everyone know what CLI means?  Perhaps replace that with "improved command line parsing"?

Reply:
Updated the wording per your comments - also changed "protocol-level handshaking protocol" to "improved handshake protocol"